### PR TITLE
#162727414 setup react router dom

### PR DESCRIPTION
**Pull request by Proception**
#### What does this PR do?
Implement router for ArtCave front end project

#### Description of Task to be completed?
- add 404 route
- add dummy login route
- add dummy home route
- add menu, login and home component
- add routes file to hold all routes

#### How should this be manually tested?
 - Run `npm i`
 - Start Development Server by running `npm run start:dev`
 - Open a browser and visit the following urls `localhost:5050/` and `localhost:5050/login`

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162727414

#### Screenshots (if appropriate)
None

#### Questions:
None

### CODE REVIEW

File: src/components/Menu.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import { NavLink } from 'react-router-dom';
+
+export default class Menu extends Component {
+  render() {
+    return (
+      <div>
+        <ul>
+          <li><NavLink to="/" >Home</NavLink></li>
```
Comment (abejide001) [inline src/components/Menu.jsx:9]: Any specific reason for using `NavLink` instead of `Link` 🤔

**Review by abejide001 — COMMENTED** [review]
Left a comment

File: src/components/Menu.jsx (hunk 1 above)
Comment (Proception) [reply src/components/Menu.jsx:9]: A special version of the `<Link>` that will add styling attributes to the rendered element when it matches the current URL.

File: src/index.js [hunk 2]
```diff
@@ -1,8 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import { BrowserRouter as Router } from 'react-router-dom'
 
 ReactDOM.render(
-  <App />,
+  <Router>
```
Comment (andela-dbamidele) [inline src/index.js]: Can you move this router to App.jsx?

Comment (Proception) [reply src/index.js]: Done

**Review by andela-dbamidele — APPROVED** [review]

**Pull request merged**